src/Conversion/ONNXToKrnl/ConvertONNXToKrnl.cpp

```diff
@@ -208,6 +208,8 @@ void populateONNXToKrnlConversionPattern(RewritePatternSet &patterns,
   populateLoweringONNXScanOpPattern(patterns, typeConverter, ctx);
   // Math
   populateLoweringONNXCumSumOpPattern(patterns, typeConverter, ctx);
+  populateLoweringONNXDFTOpPattern(patterns, typeConverter, ctx);
+  populateLoweringONNXDFTV17OpPattern(patterns, typeConverter, ctx);
```

Comment:
I think we don't need lowering DFTV17Op since it was decomposed into DFTOp. Please remove this.

Reply:
Removed